Lib/ruby/director.swg

```diff
@@ -151,6 +151,8 @@ namespace Swig {
     const char *what() const throw() {
       return swig_msg.c_str();
     }
+
+    int status;
```

Comment:
Let's not store status unless we know it is needed. It can be added in if someone requires it, then we'll have a valid use case too.